.circleci/config.yml

```diff
@@ -331,6 +331,31 @@ jobs:
             docker push "tendermint/tendermint"
             docker logout
 
+  reproducible_builds:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
```

Comment:
"We now offer Docker Layer Caching on CircleCI 2.0 for an additional fee. "

Are we paying CircleCI for this?

Reply:
No idea. CC'ing @greg-szabo

Reply:
@ebuchman can you check? I believe Greg is on vacation

Reply:
> No idea

Why did you add it then?

Reply:
If the feature is enabled, it would be used. Else it's a no-op.

Reply:
```You must open a support ticket to have a CircleCI Sales representative enable Docker Layer Caching on your circleci.com account for an additional fee.```

I doubt anyone did this as we never looked at using the feature.